homeassistant/components/lg_thinq/strings.json

```diff
@@ -43,6 +43,256 @@
+        }
+      },
+      "current_job_mode": {
+        "name": "Operating mode",
+        "state": {
+          "air_clean": "Purifying",
+          "auto": "Auto",
+          "baby_care": "Baby care mode",
+          "circulator": "Booster",
+          "clean": "Single",
+          "direct_clean": "Direct mode",
+          "dual_clean": "Daul",
+          "fast": "Turbo",
+          "heat_pump": "Heat pump",
+          "humidify": "Mist",
+          "humidify_and_air_clean": "Mist & purifying",
+          "humidity": "Humid",
+          "nature_clean": "Natural mode",
+          "pet_clean": "Pet mode",
+          "silent": "Silent",
+          "sleep": "Sleep",
+          "smart": "Smart mode",
+          "space_clean": "Diffusion mode",
+          "spot_clean": "Wide mode",
+          "turbo": "Turbo",
+          "up_feature": "Additional mode",
+          "vacation": "Vacation"
+        }
+      },
+      "operation_mode": {
+        "name": "Operation",
+        "state": {
+          "cancel": "Cancel",
+          "power_off": "Power off",
+          "preheating": "Preheating",
+          "start": "Start",
+          "stop": "Stop",
+          "wake_up": "Sleep mode off"
+        }
+      },
+      "operation_mode_for_location": {
+        "name": "{location} operation",
+        "state": {
+          "cancel": "Cancel",
+          "power_off": "Power off",
+          "preheating": "Preheating",
+          "start": "Start",
+          "stop": "Stop",
+          "wake_up": "Sleep mode off"
+        }
+      },
+      "air_clean_operation_mode": {
+        "name": "Air purify",
+        "state": {
+          "start": "Start",
+          "stop": "Stop"
+        }
+      },
+      "cook_mode": {
+        "name": "Cook mode",
+        "state": {
+          "bake": "Bake",
+          "convection_bake": "Convection bake",
+          "convection_roast": "Convection roast",
+          "roast": "Roast",
+          "crisp_convection": "Crisp convection"
+        }
+      },
+      "cook_mode_for_location": {
+        "name": "{location} cook mode",
+        "state": {
+          "bake": "Bake",
+          "convection_bake": "Convection bake",
+          "convection_roast": "Convection roast",
+          "roast": "Roast",
+          "crisp_convection": "Crisp convection"
+        }
+      },
+      "light_brightness": {
+        "name": "Light"
+      },
+      "optimal_humidity": {
+        "name": "Ventilation",
+        "state": {
+          "on": "On",
+          "off": "Off"
+        }
+      },
+      "hot_water_mode": {
+        "name": "Hot water",
+        "state": {
+          "on": "On",
+          "off": "Off"
+        }
+      },
+      "warm_mode": {
+        "name": "Heating",
+        "state": {
+          "warm_on": "On",
+          "warm_off": "Off"
+        }
+      },
+      "humidity_warm_mode": {
+        "name": "Warm mist",
+        "state": {
+          "warm_on": "On",
+          "warm_off": "Off"
+        }
+      },
```

Comment:
These all sound like they should be switches instead

Reply:
If we move them to the siwtch, 
'header toggle' switch to control all together is enabled automatically.
Can we disable it? Or we want to keep these here.

Reply:
What's header toggle?

Reply:
If there are more than one switch, a switch that control everything is enabled.
It is a header toggle.

Reply:
So how is that enabled then? We choose what entities we want for what device right?

Reply:
Of course, we can control each.
But those switches are not irrelevant.
So it is quite awkward to control them with a 'header toggle'.
We don't want that UX.

Reply:
I think I'm not completely understanding. These selects are now used for toggling features of a device.

You're talking about a switch that can turn off everything?

Where does that header toggle come from? (Where does the name come from? I'm assuming it's a toggle in a header, but nothing comes to mind when thinking of that, so if you have a screenshot of the problem, yes please)

Reply:
Problem>
![Screenshot_20240927_155615_Home Assistant](https://github.com/user-attachments/assets/ff1a7c88-e25a-4a32-893d-4e5c40f2a18b)

Our intention>
![Screenshot_20240927_155559_Home Assistant](https://github.com/user-attachments/assets/a4970d13-4737-46fb-9f50-e1a3af4ec40b)

Reply:
The related front -end code is as below.
https://github.com/home-assistant/frontend/blob/c30b9cdfcfa632610f940a32921d8040092a497f/src/panels/lovelace/cards/hui-entities-card.ts#L120

Reply:
I also discussed this one and we came to the following conclusions:
1. A humidifier is better represented by a [humidifier entity](https://developers.home-assistant.io/docs/core/entity/humidifier)
2. You can set the `entity_category` of those entities to `EntityCategory.CONFIG` and it will not show up on the default dashboard
3. This is mostly a frontend thing. We should not change choices in the backend because the frontend handles it differently.

Does this help?

Reply:
> I also discussed this one and we came to the following conclusions:
> 
> 1. A humidifier is better represented by a [humidifier entity](https://developers.home-assistant.io/docs/core/entity/humidifier)
> 2. You can set the `entity_category` of those entities to `EntityCategory.CONFIG` and it will not show up on the default dashboard
> 3. This is mostly a frontend thing. We should not change choices in the backend because the frontend handles it differently.
> 
> Does this help?

As we tested, humidifier entity doesn't support steps,
and our devices only support 5 steps.
It causes errors whenever a user set humidifier target value.
That's why we didn't use humidifier entity.

Reply:
Yep, please discuss this internally. I think it's good to know not a lot of users are using the default dashboard, so it probably won't affect that much users. Also, a new updated default dashboard is on the roadmap, so eventually it will be better (I don't think that they are planning to use that frontend card, so that means no header toggle)

Reply:
> Yep, please discuss this internally. I think it's good to know not a lot of users are using the default dashboard, so it probably won't affect that much users. Also, a new updated default dashboard is on the roadmap, so eventually it will be better (I don't think that they are planning to use that frontend card, so that means no header toggle)

I just want to make sure what you want.
1) Not using humidifier entity is acceptable
2) Changing entities to CONFIG is strongly recommended.

Am I right?

Reply:
> As we tested, humidifier entity doesn't support steps,
and our devices only support 5 steps.
It causes errors whenever a user set humidifier target value.

So you mean you have like 0%, 20%, 40%, 60%, 80% or something along those lines as modes to set the target humidity to?

Reply:
> > As we tested, humidifier entity doesn't support steps,
> > and our devices only support 5 steps.
> > It causes errors whenever a user set humidifier target value.
> 
> So you mean you have like 0%, 20%, 40%, 60%, 80% or something along those lines as modes to set the target humidity to?

Yes, that's correct.

Reply:
All switch-like entities are removed from 'select' platform.
Let me add those entities into 'switch' platform with a new PR later.